### DISCUSSION TITLE
链式代理是在哪里配置？

### DISCUSSION THREAD

**Discussion by zwl1619** [MetaCubeX/metacubexd#749] [Q&A]
链式代理是在哪里配置？
已经买了一个vps，搭好了服务端，可以正常运行。现在又买了个ip代理，它给了我一个IP地址以及用户名密码，我想用这个ip代理作为落地，配置文件要怎么添加？
我看了一下文档配置页的几个部分，https://wiki.metacubex.one/config/ ，但没找到相关内容是在哪里。
大佬指导一下。

**Comment by Plutonium141** [comment]
在这里 [https://wiki.metacubex.one/config/proxies/dialer-proxy/](https://wiki.metacubex.one/config/proxies/dialer-proxy/)
